data/lua/stun.lua

```diff
@@ -0,0 +1,72 @@
+
```

Comment:
Looks like we need to fix a bug in the fix-whitespace script, because the CI should have caught this.

Reply:
???

Reply:
One of our tools fixes blank lines at the end of the file, and two consecutive blank lines in the middle of files. Maybe it only runs on .cfg files.

Reply:
But this is neither of those?

Reply:
Having checked the old discussion, okay blank first lines are accepted.